platform/pulp/platform/models/consumer.py

```diff
@@ -0,0 +1,67 @@
+"""
+Django models related to content consumers.
+"""
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db.models import TextField, ForeignKey, ManyToManyField, CASCADE
+
+from pulp.platform.models import Model, Notes
+from pulp.platform.models.repository import Distributor
+
+
+class Consumer(Model):
+    """
+    A content consumer.
+
+    Fields:
+
+    :cvar name: The consumer common name.
+    :type name: TextField
+
+    :cvar: description: An optional description.
+    :type: TextField
+
+    Relations:
+
+    :cvar notes: Arbitrary information about the consumer.
+    :type notes: GenericRelation
+
+    :cvar distributors: Associated distributors.
+    :type distributors: ManyToManyField
+    """
```

Comment:
You'd mentioned documenting cvars and ivars explicitly, is that still something you plan to do, or did it not work?